jbcli/jbcli/tests/test_docker.py

```diff
@@ -16,8 +16,8 @@ def test_up(self, check_mock):
         dockerutil.up()
         assert check_mock.mock_calls == [
             call(['docker-compose',
-                  '--project-directory', '.', '--project-name', 'stable',
-                  '-f', '../docker-compose.common.yml', '-f', 'docker-compose.yml',
+                  '--project-directory', '.', '--project-name', 'devlandia',
+                  '-f', 'docker-compose.common.yml', '-f', 'docker-compose.yml',
```

Comment:
now that we only have one docker-compose.yml, we can drop docker-compose.common.yml and just include it directly into docker-compose.yml